### PR TITLE
Route Code Splitting

### PR DESCRIPTION
**Pull request by jvonasek**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,15 +3,44 @@ import {
...
+const AppProviders = lazy(async () => ({
+  default: (await import("components/AppProviders/AppProviders")).AppProviders,
+}))
+
 const history = createBrowserHistory()
 const location = new ReactLocation({ history })
 
+const HydraSplash = () => {
+  return (
+    <div
```
Comment (vkulinich-cl) [inline src/App.tsx:23]: can you create a separate styled component?

File: src/components/Layout/Header/banners/HeaderBanners.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { WarningMessage } from "components/WarningMessage/WarningMessage"
```
Comment (vkulinich-cl) [inline src/components/Layout/Header/banners/HeaderBanners.tsx:1]: is that a new component? migration was done in the separate PR, right?

Comment (jvonasek) [reply src/components/Layout/Header/banners/HeaderBanners.tsx:1]: HeaderBanners is new component so all banners can be lazy loaded.

Migration banner is also in this new component.

File: src/App.tsx (hunk 1 above)
Comment (jvonasek) [reply src/App.tsx]: done

**Review by vkulinich-cl — APPROVED** [review]
